.vscode/settings.json

```
@@ -2125,7 +2125,262 @@
+                        "# フィールド",
+                        "    # data modify storage asset:object Field.myValue set value "
+                    ]
+                },
+                {
+                    "type": "file",
+                    "rel": "data/asset/functions/object/alias/%id%/register.mcfunction",
+                    "content": [
+                        "#> asset:object/alias/%id%/register",
+                        "#",
+                        "# Objectのデータ指定処理のエイリアス",
+                        "#",
+                        "# @within asset_manager:object/summon/summon.m",
+                        "",
+                        "# 元の登録処理を呼び出す",
+                        "   function asset:object/%namespace%/register"
+                    ]
+                },
+            ],
+            "customQuestion": [
+                {
+                    "name": "id",
+                    "question": "ObjectのID",
+                    "pattern": "[1-9][0-9]*",
+                    "patternErrorMessage": "数値のみが許されるよ"
+                }
+            ]
+        },
+        {
+            "label": "Object/ 召喚処理",
+            "generates": [
+                {
+                    "type": "file",
+                    "rel": "data/asset/functions/object/%namespace%/summon/.mcfunction",
+                    "content": [
+                        "#> asset:object/%namespace%/summon/",
+                        "#",
+                        "# Object召喚処理の呼び出し時に実行されるfunction",
+                        "#",
+                        "# @within asset:object/alias/%id%/summon",
+                        "",
+                        "# 元となるEntityを召喚する",
+                        "    summon %entity% ~ ~ ~ {Tags:[\"ObjectInit\"]}",
+                        "",
+                        "# 実装フラグを立てる",
+                        "data modify storage asset:object Implement set value true"
+                    ]
+                },
+                {
+                    "type": "file",
+                    "rel": "data/asset/functions/object/alias/%id%/summon.mcfunction",
+                    "content": [
+                        "#> asset:object/alias/%id%/summon",
+                        "#",
+                        "# Object召喚処理のエイリアス",
+                        "#",
+                        "# @within asset_manager:object/summon/summon.m",
+                        "",
+                        "# 元の召喚処理を呼び出す",
+                        "   function asset:object/%namespace%/summon/"
+                    ]
+                },
+                {
+                    "type": "file",
+                    "rel": "data/asset/functions/object/%namespace%/summon/debug.mcfunction",
+                    "content": [
+                        "#> asset:mob/%namespace%/summon/debug",
+                        "#",
+                        "# 動作チェック用の召喚処理 使い終わったら消してもいいかも",
+                        "#",
+                        "# @within asset:object/%namespace%/summon/debug",
```

nits: 
- IMPDoc をこうするとよいです
- ｢summon は継承先のを使いたいけどこいつの召喚用の debug ファンクションはほしい｣ ってニーズが結構ありそうなので、こいつは ｢Object/登録処理｣ に入れても良いかも
- それはそうとこれを自動生成してくれるのは良いですネ
```suggestion
                        "#> asset:mob/%namespace%/summon/debug",
                        "#",
                        "# 動作チェック用の召喚処理 使い終わったら消してもいいかも",
                        "#",
                        "# @private",
                        "# @user",
```